Model/Model.tex

```diff
@@ -826,11 +826,21 @@ \subsection{Global variables}
 Global variables are accessible everywhere. 
 Every class is named by a global variable; in addition, a few globals are used to name special or commonly useful objects.
 
-The variable \glbind{Transcript} names an instance of \clsind{TranscriptStream}, a stream that writes to a scrolling window.
-The following code displays some information and then goes to the next line in the \ct{Transcript}.
+However, what may sound so handy and convenient at the first glance has turned out as bad practice during the evolution of \sq.
+While we state in rule 4 that ``everything happens by message sends'', the concept of globals is a violation of this rule.
+Accessing a global allows you to exchange information between two objects\,---\,\emph{without} sending any message.
+As a consequence, when using globals, we loose an important advantage of the \st object model, which is the total encapsulation of every object's state.
+Also, global variables introduce \emph{global state} as mentioned earlier which impedes the modularity of \st objects.
```

Comment:
maybe add something like "... and hinders us to use a \st object directly without taking care of the context."

"context" does not really sound right yet, but something that describes the actual consequence for users would be good here. "Modularity" is a rather abstract thing.

Reply:
What about "environment" instead of "context"?

Reply:
Yes, "environment" would work :)